src/cli_agent_orchestrator/services/wiki_compiler.py

```
@@ -0,0 +1,860 @@
+
+# Test seam: monkeypatch this to inject mock LLM behaviour without touching
+# the production path. Signature: ``async (client, system, user, *, timeout_s) -> str``.
+_llm_call: Callable[..., Awaitable[str]] = _default_llm_call
+
+
+# -----------------------------------------------------------------------------
+# Public entry point
+# -----------------------------------------------------------------------------
+
+
+async def compile(
```

Ruff (0.15.17)

[error] Variable `compile` is shadowing a Python builtin

(A001)